### PR TITLE
Gp 10484/fix ticket and pr env vars

### PR DESCRIPTION
**Pull request by dcumberland**
The v3.0.4 template has a couple of errors with respect to Ticket and PR processing configs.
This update will have PR processing only run in the PR pod.
This update will have ticket processing only run in the ticket pod.

### CODE REVIEW

File: charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+              value: {{ quote .templateData.ticketWebhookListeners }}
+          {{- end}}
+          {{- if .templateData.ticketWebhookThreadsPerListener }}
+            - name: GP_TICKET_WEBHOOK_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.ticketWebhookThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.repoDeleteListeners }}
+            - name: GP_REPO_DELETE_LISTENERS
+          value: {{ quote .templateData.repoDeleteListeners }}
+          {{- end}}
```
Comment (bajolly) [inline charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl:127]: Not sure if indention matters here with the yaml.

File: charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+          {{- if .templateData.repoDeleteThreadsPerListener }}
+            - name: GP_REPO_DELETE_PER_LISTENER
+          value: {{ quote .templateData.repoDeleteThreadsPerListener }}
+          {{- end}}
```
Comment (bajolly) [inline charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl:131]: Indention

**Review by bajolly — APPROVED** [review]
A few comments about indention but nothing blocking assuming the indention doesn't matter.

File: charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl (hunk 2 above)
Comment (dcumberland) [reply charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl:131]: Great catch! 👍

File: charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl (hunk 1 above)
Comment (dcumberland) [reply charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl:127]: Great catch! 👍

**Pull request merged**